README.md

```diff
@@ -1,73 +1,22 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# Hello-microverse
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly replace the `Major languages` with `HTML`, and `Frameworks` with `CSS`, and remove the `Technologies` to make the readme well arranged to look more professional.